CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog 
 
+## Version 2.9.0-beta3
+- Update Base SDK to version 2.9.0-beta3
+
 ### Version 2.9
```

Comment:
since we are shipping 2.9.0-beta3, lets correct te changelog to say 2.9.0-beta3. (instead of 2.9)